spec/controllers/api/v4/runs_controller_spec.rb

```diff
@@ -175,7 +175,7 @@
       it 'returns the correct content-type header' do
         request.headers['Accept'] = 'application/wsplit'
 
-        expect(response.content_type).to eq('application/wsplit')
+        expect(response.media_type).to eq('application/wsplit')
```

Comment:
`response.content_type` now returns the full content type header as is, while `response.media_type` now returns just the MIME component as previous versions of rails did.